rst_files/julia_essentials.rst

```diff
@@ -974,29 +990,30 @@ While the above was convenient, there are other times when you want to simply fi
 
 .. code-block:: julia
 
-    # f2(x) = q * x^2 # fails during compilation.  `q` is not defined
     a = 0.2
-    f(x) = a * x^2 # refers to the `a` in the outer scope
-    f(1) # univariate function
+    f(x) = a * x^2     # Refers to the `a` in the outer scope
+    f(1)               # Univariate function
 
-When the function ``f`` is parsed in julia, it will look to see if any of the variables are already defined in the current scope
+When the function ``f`` is parsed in Julia, it will look to see if any of the variables are already defined in the current scope
 
-In this case, it finds the ``a`` since it was defined previously, whereas if the code defined ``a = 0.2`` **after** the `f(x)` definition, it would fail
+In this case, it finds the ``a`` since it was defined previously, whereas if the 
+code defined ``a = 0.2`` **after** the ``f(x)`` definition, it would fail
 
 This also works when embedded in other functions
 
 .. code-block:: julia
 
     function g(a)
-        f(x) = a * x^2 # refers to the `a` passed in the function
-        f(1) # univariate function
+        f(x) = a * x^2  # Refers to the `a` passed in the function
+        f(1)            # Univariate function
     end
     g(0.2)
 
 Comparing the two:  the key here is not that ``a`` is a global variable, but rather that the ``f`` function is defined to capture a variable from an outer scope
 
 This is called a **closure**, and are used throughout the lectures
 
+**what does this mean**
```

Comment:
Changed this to `captured variable`, which is what I think was meant. Also, important to make sure comments are left as genuine rST comments, since output like this can sneak through.

Reply:
(That is, if comments are left at all, per the other PR).

Reply:
Wait, you got rid of **closure**?  That is an important word to teach people.  A closure also "captures a variable", it isn't a captured variable?

Reply:
No, didn't get rid of that section. Just changed "closure variable" to "captured variable" in: 

```
This is called a **closure**, and are used throughout the lectures

It is generally bad practice to modify the closure variable in the function, but otherwise the code becomes very clear

One place where this can be helpful is in a string of dependent calculations
```

Reply:
So you changed it from "modify the closure variable in the function" to "modify the captured variable in the function"?  Sounds good.

Reply:
Exactly.